src/main/java/sirius/biz/analytics/events/EventRecorder.java

```diff
@@ -227,6 +229,44 @@ public <E extends Event> int countEventsInRange(Class<E> eventType,
         });
     }
 
+    /**
+     * Counts the number of events which have occurred in the last month based on the given <tt>queryTuner</tt>.
+     * <p>
+     * This automatically marks the query as long-running.
+     *
+     * @param eventType the type of events to query
+     * @param queryTuner the actual filter to apply
+     * @return the number of events matching the given filter in the last month. Note that we return an <tt>int</tt>
+     * here to better match the API of {@link sirius.kernel.health.metrics.Metrics}.
+     * @param <E> the generic types of the entities to query
+     * @throws SQLException in case of a database error
+     */
+    public <E extends Event> int countEventsInLastMonth(Class<E> eventType, @Nullable Consumer<SmartQuery<E>> queryTuner) throws SQLException {
+        YearMonth lastYearMonth = YearMonth.now().minusMonths(1);
+        LocalDateTime startDate = lastYearMonth.atDay(1).atStartOfDay();
+        LocalDateTime endDate = lastYearMonth.atEndOfMonth().atTime(LocalTime.MAX);
+        return countEventsInRange(eventType, startDate, endDate, queryTuner);
+    }
+
+    /**
+     * Counts the number of events which have occurred in the last year based on the given <tt>queryTuner</tt>.
+     * <p>
+     * This automatically marks the query as long-running.
+     *
+     * @param eventType the type of events to query
+     * @param queryTuner the actual filter to apply
+     * @return the number of events matching the given filter in the last year. Note that we return an <tt>int</tt>
+     * here to better match the API of {@link sirius.kernel.health.metrics.Metrics}.
+     * @param <E> the generic types of the entities to query
+     * @throws SQLException in case of a database error
+     */
+    public <E extends Event> int countEventsInLastYear(Class<E> eventType, @Nullable Consumer<SmartQuery<E>> queryTuner) throws SQLException {
```

Comment:
**Question:** Did you compare this with the range logic of the `Statistics::getYearStatistics` ?